app/models/agent.rb

```diff
@@ -149,15 +153,21 @@ def update_event_expirations!
     end
   end
 
-  def trigger_web_request(params, method, format)
+  def trigger_web_request(request)
+    params = request.params.except(:action, :controller, :agent_id, :user_id, :format)
     if respond_to?(:receive_webhook)
```

Comment:
I mean, should we still offer this backward compatibility?  It's been there with a deprecation warning for years, so I thought it might be the time to get rid of it.